snippets/logging-log-types-snip.adoc

```diff
@@ -0,0 +1,15 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+The {logging-title} collects container logs and node logs. These are categorized into types:
+
+* `application` - Container logs generated by non-infrastructure containers.
+
+* `infrastructure` - Container logs from namespaces `kube-*` and `openshift-*`, and node logs from `journald`.
+
+* `audit` - Node logs from `/var/log/audit.log` and `/var/log/openshift-apiserver/audit.log`.
```

Comment:
ovn logs... i don't know from where they originate and kube audit logs at `/var/log/kube-apiserver/audit.log`